tests/catalog/test_glue.py

```diff
@@ -707,6 +707,7 @@ def test_commit_table_update_schema(
     test_catalog.create_namespace(namespace=database_name)
     table = test_catalog.create_table(identifier, table_schema_nested)
     original_table_metadata = table.metadata
+    last_update_ms = original_table_metadata.last_updated_ms
```

Comment:
nit: not necessary to check `last_update_ms` for all updates. perhaps just when unit testing `update_table_metadata`

Reply:
Agreed, will have have 1 or 2 assertions in `test_glue` and `test_sql`.